dffml/source/file.py

```diff
@@ -114,11 +115,16 @@ def args(cls, args, *above) -> Dict[str, Arg]:
             "readonly",
             Arg(type=bool, action="store_true", default=False),
         )
+        cls.config_set(args,
+                       above,
+                       "src_url",
```

Comment:
Let's name this `key` instead of `src_url`

Reply:
okay :+1: